### PR TITLE
Plot graph documentation extending [part of #180]

### PR DESCRIPTION
**Pull request by mikhailnikolaev**
This PR contains a part of changes from the closed PR #180. It is the seventh part of the list, described here https://github.com/awf/ADBench/pull/180#issuecomment-569066919

### CODE REVIEW

File: docs/PlotCreating.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Special plotting script is responsible for creating benchmark plots. It is a Pyt
 
 These dependencies are fetched automatically during CMake configure.
 
-This script looks through the directory `tmp` and for every subdirectory, that contains results produced by [the global runner](./GlobalRunner.md), it creates graphs and stores them in the folder `tmp/graphs`. Two kinds of graphs are available: static PNG files and dynamic [plotly](https://plot.ly/python/) HTML files.
+By default, this script looks through the directory `tmp` and for every its subdirectory, that contains results produced by [the global runner](./GlobalRunner.md), it creates graphs and stores them into the folder `tmp/graphs`. Two kinds of graphs are available: static PNG files and dynamic [plotly](https://plot.ly/python/) HTML files. Also, it saves two JSON files that are contain all test sizes (`graphs_index.json`) and all collected plotted info, e.g. times, tool names etc. (`plot_data.json`). If
```
Comment (tomjaguarpaw) [inline docs/PlotCreating.md]: Actually absence of "its" and "in" rather than `into` were already correct here.

File: docs/PlotCreating.md [hunk 2]
```diff
@@ -20,13 +20,14 @@ The script command line arguments are:
 | `--save` | Makes the script save the static PNG files with the created plots |
 | `--plotly` | Makes the script save the dynamic plotly HTML files with the created plots |
 | `--show` | Makes the script show the created static plots on the screen |
+| `--use-file <file>` | Makes the script use the specified input file, holds plot data information, instead of data files generated by the global runner. Also, the script will save all outputs to the directory where the `file` is located. |
```
Comment (tomjaguarpaw) [inline docs/PlotCreating.md]: "*which* holds plot data information`

File: docs/PlotCreating.md (hunk 1 above)
Comment (mikhailnikolaev) [reply docs/PlotCreating.md]: Reverted

File: docs/PlotCreating.md (hunk 2 above)
Comment (mikhailnikolaev) [reply docs/PlotCreating.md]: Fixed

**Pull request merged**